Makefile

```diff
@@ -153,7 +147,7 @@ all-srcs:
 
 .PHONY: cover
 cover: nocover
-	$(GOTEST) -coverprofile cover.out ./...
+	$(GOTEST) -timeout 5m -coverprofile cover.out ./...
```

Comment:
default timeout is 10min, which is the same as Travis's no-output timeout, so Travis kills the job before go test has a chance to timeout and print the stack dump if there is a deadlock.